### ISSUE TITLE
Add some sort of `Spring.SendChatMessage`

### ISSUE THREAD

**Issue by sprunk** [beyond-all-reason/spring#1582]
Right now you have to manually invoke the `/say` command via `Spring.SendCommands`, and for ally/spectator chat you gotta append the `a:` or `s:` manually as well.

Add explicit functions, for example `SendChatMessage`, `SendAllyChatMessage` and `SendSpectatorChatMessage`.

**Comment by SamWain** [comment]
This has been completed hasn't it?

**Comment by sprunk** [comment]
I don't know, can you point to the functions?

**Comment by SamWain** [comment]
[static int SendMessage(lua_State* L)](https://github.com/beyond-all-reason/spring/blob/625902d539f43871ceb4ecdc45fc539e91a71b55/rts/Lua/LuaUnsyncedCtrl.h#L22)
[LuaUnsyncedCtrl::SendMessage](https://github.com/beyond-all-reason/spring/blob/625902d539f43871ceb4ecdc45fc539e91a71b55/rts/Lua/LuaUnsyncedCtrl.cpp#L580)

**Comment by sprunk** [comment]
No, that's just a fancy version of `Spring.Echo`. The ticket asks for chat messages.

**Comment by badosu** [comment]
What about a unified interface for calling out [synced/unsynced commands](https://beyond-all-reason.github.io/spring/guides/unsynced-commands/) instead? This would take benefits of typing and support the overall area more efficiently (without the need for too many explicit callins, one for each command).

Example:

```lua
local sayArguments = { playerId = 123, message = "Hue" } --- @type UnsyncedCmd.Say.Args
Spring.SendCommand(UnsyncedCmd.Say, sayArguments)
```

The "command" terminology is just problematic though
